app/styles/pages/admin.scss

```diff
@@ -0,0 +1,3 @@
+.sixteen.wide.column {
```

Comment:
Now all sixteen wide columns everywhere will have this style. We only want it for selected divs. Create a new class may be called as `x-scrollable` and add it there.